src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -39,6 +39,26 @@ std::string UnorderedMapToString(const std::unordered_map<std::string, double> &
   return buffer.str();
 }
 
+/// Convert a vector of doubles to a vector of resource units.
```

Comment:
Can we remove the below two functions and not use `std::vector<double>` for resources at all?